proposals/lip-0049.md

```diff
@@ -358,12 +358,12 @@ def forward(trs: Transaction, ccm: CCM) -> None:
         ccmProcessedEventCode = CCM_PROCESSED_CODE_CHANNEL_UNAVAILABLE
         
     # If the chain is registered, do not continue.
-    if chainAccount(ccm.receivingChainID).status == CHAIN_STATUS_REGISTERED:
+    elif chainAccount(ccm.receivingChainID).status == CHAIN_STATUS_REGISTERED:
```

Comment:
in python we could also just do `if chainAccount(ccm.receivingChainID) does not exist or chainAccount(ccm.receivingChainID).status == CHAIN_STATUS_REGISTERED:` but this is a pure cosmetic comment so up to you

Reply:
Done like you suggested, looks much better.